set.mm

```diff
@@ -50271,10 +50271,10 @@ holding in an empty domain (see Axiom A5 and Rule R2 of [LeBlanc]
   ${
     $d w x y z $.
     $( Alternate proof of ~ dtru using ~ ax-pow instead of ~ ax-pr .
-       (Contributed by NM, 7-Nov-2006.)  Avoid ~ ax-13 .  (Revised by Gino
-       Giotto, 5-Sep-2023.)  Avoid ~ ax-12 .  (Revised by Rohan Ridenour,
-       9-Oct-2024.)  (Proof modification is discouraged.)
-       (New usage is discouraged.) $)
+       (Contributed by NM, 7-Nov-2006.)  Avoid ~ ax-13 .  (Revised by BJ,
+       31-May-2019.)  (Revised by Gino Giotto, 5-Sep-2023.)  Avoid ~ ax-12 .
```

Comment:
I remember somewhere that you removed ax-13 before me, so you can remove my credit.

Reply:
I remember you had also removed ax-13 later and independently (since bj-dtru was in my mathbox, it was less visible). Following your comment, I am removing the tag concerning your revision.